akd/agents/data_search/components/cmr_query_generation.py

```diff
@@ -183,6 +189,81 @@ def _summarize_query(self, query) -> str:
 
         return ", ".join(parts) if parts else "empty query"
 
+    def _create_fallback_queries(
+        self,
+        scientific_angle: ScientificAngle,
+        original_query: str,
+    ) -> CMRQueryOutput:
+        """Create fallback CMR queries when LLM generation fails."""
+        logger.warning("Using fallback CMR queries due to LLM failure")
+
+        # Extract basic keywords from the scientific angle and original query
+        keywords = []
+
+        # Parse angle title for keywords
+        angle_words = scientific_angle.title.lower().split()
+        keywords.extend([word for word in angle_words if len(word) > 3])
+
+        # Parse original query for keywords
+        query_words = original_query.lower().split()
+        keywords.extend([word for word in query_words if len(word) > 3])
+
+        # Remove common stop words and duplicates
+        stop_words = {
```

Comment:
Probably get this from config and default to these values. That way we can externally change stopwords